### PR TITLE
Enable new notprod domain

### PR DESCRIPTION
**Pull request by laurieboyes**
Was going to do a straight switch but saw an opportunity to do it the non-breaking way

Stephen’s PR turns the URL into a variable, which is of course incompatible with this change: https://github.com/UKHomeOffice/callisto-auth-keycloak/pull/8

My plan is to get this going and then switch back to just the new URL afterwards

Bonus: some readme additions

### CODE REVIEW

**Review by JWeston-HO — APPROVED** [review]

**Review by daleelR — APPROVED** [review]

**Comment by laurieboyes** [comment]
Tried this locally and it broke stuff. Marked as WIP while we fix it

File: config/react-client.tf [hunk 1]
```diff
@@ -15,13 +15,13 @@ resource "keycloak_openid_client" "react_client" {
   direct_access_grants_enabled             = true
   service_accounts_enabled                 = false
 
-  admin_url = "https://web.dev.callisto.homeoffice.gov.uk/"
-  root_url  = "https://web.dev.callisto.homeoffice.gov.uk/"
```
Comment (laurieboyes) [inline config/react-client.tf:19]: It appears to me that we’re not actually using these:
https://registry.terraform.io/providers/mrparkers/keycloak/latest/docs/resources/openid_client#admin_url
https://registry.terraform.io/providers/mrparkers/keycloak/latest/docs/resources/openid_client#root_url

File: README.md [hunk 2]
```diff
@@ -1,10 +1,56 @@
...
+  - The terraform CLI (install with brew)
+  - The AWS CLI
+- access to our k8s secrets to see the `ho-callisto-terraform-s3` secrets for configuring the back-end
+- ‘ream-management’ client roles in keycloak for the callisto-dev realm (a colleague should be able to set you up here)
+
+Steps:
+
+1. `cd config`
+
+1. `terraform init`
```
Comment (daleelR) [inline README.md:29]: nit: this and the following step numbers are all '1'.

Comment (daleelR) [reply README.md:29]: my bad, it's just the non formatted view

**Pull request merged**